metadata/src/audio/item.rs

```diff
@@ -16,98 +17,255 @@ use librespot_core::{
 
 pub type AudioItemResult = Result<AudioItem, Error>;
 
-// A wrapper with fields the player needs
 #[derive(Debug, Clone)]
-pub struct AudioItem {
-    pub id: SpotifyId,
-    pub spotify_uri: String,
-    pub files: AudioFiles,
-    pub name: String,
-    pub duration: i32,
-    pub availability: AudioItemAvailability,
-    pub alternatives: Option<Tracks>,
-    pub is_explicit: bool,
+pub enum AudioItem {
+    Track(Box<Track>),
+    Episode(Box<Episode>),
```

Comment:
Is there a specific reason, this has to be boxed? It seems to compile just fine without.

Reply:
Clippy was complaining that they should be boxed.

Reply:
Ah, right. Then this makes sense, of course!

Reply:
All hail the clippy overlord,lol!!!